reducers/views/threads.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {combineReducers} from 'redux';
+
+import {GenericAction} from 'mattermost-redux/types/actions';
+
+import {ViewsState} from 'types/store/views';
+
+import {Threads} from 'utils/constants';
+
+export const selectedThreadIdInTeam = (state: ViewsState['threads']['selectedThreadIdInTeam'] | null = null, action: GenericAction) => {
+    if (action.type === Threads.CHANGED_SELECTED_THREAD) {
```

Comment:
nit: can we use the `switch` block despite needing to only handle one action type?